src/test-run/index.js

```diff
@@ -34,12 +53,13 @@ export default class TestRun extends Session {
 
     async _start () {
         this.started = true;
+        this.emit('start');
 
         try {
             await this.test.fn(this);
         }
         catch (err) {
-            this.errs.push(err);
+            this.errs.push(new TestRunErrorFormattableAdapter(err, this.browserConnection.userAgent, '', ''));
```

Comment:
`executeCommand` should accept `callsite` as additional argument and store it with pending command. So if error occurs we will take callsite from the pending command and pass it to adapter.